test/unit/range/view/view_to_lower_test.cpp

```diff
@@ -0,0 +1,77 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
+// -----------------------------------------------------------------------------------------------------
+
+#include <iostream>
+
+#include <gtest/gtest.h>
+
+#include <seqan3/alphabet/nucleotide/dna5.hpp>
+#include <seqan3/range/concept.hpp>
+#include <seqan3/range/view/to_char.hpp>
+#include <seqan3/range/view/to_lower.hpp>
+#include <seqan3/std/ranges>
+
+using namespace seqan3;
+
+TEST(view_to_lower, basic)
+{
+
+    std::string input_string {"IAmADnaString"};
+    std::string cmp_string {"iamadnastring"};
+    std::string cmp2{"gnirtsandamai"};
+
+    std::vector<dna5> dna_vec {"AGGCGT"_dna5};
+    std::string cmp3{"aggcgt"};
+
+    std::vector<std::string> input_vec{"IAmADnaString", "IAmAProteinString"};
```

Comment:
Can you split this test into several smaller ones?

If something goes wrong it is easier to see in the googletest framework.
I would distinguish between basic, basic_deep and piping/combinability